typings/browser.d.ts

```diff
@@ -1 +0,0 @@
-/// <reference path="browser/ambient/node/index.d.ts" />
```

Comment:
Personally I am not typescript user... so I am not quite sure how to do it right way... 
Can you generate those typings?